src/components/TabTraining.vue

```diff
@@ -80,6 +87,7 @@ export default {
       saveConfig(deterministic.name, isDeterministic.value)
     })
     return {
+      ...training,
```

Comment:
What means `...training` and why we need to return it ?

Reply:
@vfdev-5 Basically training return all options in `metadata.json` and so we need this for the Training options rendering